pkgs/applications/virtualization/open-vm-tools/default.nix

```diff
@@ -23,11 +23,6 @@ stdenv.mkDerivation rec {
   buildInputs = [ fuse glib icu libdnet libmspack libtirpc openssl pam procps rpcsvc-proto xercesc ]
       ++ lib.optionals withX [ gdk-pixbuf-xlib gtk3 gtkmm3 libX11 libXext libXinerama libXi libXrender libXrandr libXtst ];
 
-  patches = [
-    ./recognize_nixos.patch
```

Comment:
This patch can't be dropped as it isn't upstreamed yet.

Reply:
Well... actually it works flawlessly. I'm using it for 2 weeks without any issues.
Earlier versions of `open-vm-tools` needed this patch and could be built without it.

I think, the reason is, that newer version of `open-vm-tools` now parse [/etc/os-release](https://github.com/vmware/open-vm-tools/blob/d0b7c90917f4f2afcb0583988d58bc3ec75ad023/open-vm-tools/lib/misc/hostinfoPosix.c#L1614) and used the infos contained therein.